src/Azs.ContainerRegistry.Admin/examples/Set-AzsContainerRegistryConfiguration.md

```diff
@@ -0,0 +1,14 @@
+### Example 1: Set Azs ContainerRegistry Configuration
+```powershell
+PS C:\> Set-AzsContainerRegistryConfiguration -MaximumCapacityInGib 30 -ConfigurationName default
```

Comment:
rename parameter to MaximumCapacityInGB

Reply:
We are following the same pattern as Azure do measuring in GiB instead of Gb